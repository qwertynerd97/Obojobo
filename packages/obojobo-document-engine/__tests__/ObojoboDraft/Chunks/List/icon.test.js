import React from 'react'
import renderer from 'react-test-renderer'

import Icon from 'ObojoboDraft/Chunks/List/icon'

describe('List Icon', () => {
	test('Icon', () => {
		const component = renderer.create(<Icon />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
