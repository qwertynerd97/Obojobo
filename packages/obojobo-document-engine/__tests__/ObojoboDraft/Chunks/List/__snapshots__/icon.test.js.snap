// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Icon Icon 1`] = `
<svg
  data-name="Layer 1"
  id="Layer_1"
  viewBox="0 0 15.04 14.32"
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <style>
      .cls-1{opacity:0.55;}.cls-2{fill:#231f20;}
    </style>
  </defs>
  <title>
    list
  </title>
  <g
    className="cls-1"
  >
    <circle
      cx="1.48"
      cy="1.48"
      r="1.48"
    />
    <circle
      cx="1.48"
      cy="7.16"
      r="1.48"
    />
    <circle
      cx="1.48"
      cy="12.84"
      r="1.48"
    />
    <rect
      className="cls-2"
      height="0.84"
      width="10.54"
      x="4.51"
      y="1.06"
    />
    <rect
      className="cls-2"
      height="0.84"
      width="10.54"
      x="4.51"
      y="6.74"
    />
    <rect
      className="cls-2"
      height="0.84"
      width="10.54"
      x="4.51"
      y="12.42"
    />
  </g>
</svg>
`;
